Dockerfile

```diff
@@ -10,7 +10,8 @@ ADD src/openshift /usr/src/app/src/openshift
 ADD src/cryostat-web /usr/src/app/src/cryostat-web
 RUN (command -v corepack || npm install --global corepack) && \
     corepack enable
-RUN npm install && yarn install && yarn build
+RUN echo "nodeLinker: node-modules" > .yarnrc.yml
```

Comment:
`.yarnrc.yml` is check in to the repo and has this line in it. Should we just `ADD` it in to this build stage?

Reply:
Hmm, it doesn't seem that simple:

```
[1/3] STEP 10/10: RUN yarn install && yarn build
! Corepack is about to download https://repo.yarnpkg.com/3.6.3/packages/yarnpkg-cli/bin/yarn.js
Internal Error: The "yarn-path" option has been set (in /usr/src/app/.yarnrc.yml), but the specified location doesn't exist (/usr/src/app/.yarn/releases/yarn-3.6.3.cjs).
    at i (/opt/app-root/src/.cache/node/corepack/v1/yarn/3.6.3/yarn.js:446:11180)
    at async t (/opt/app-root/src/.cache/node/corepack/v1/yarn/3.6.3/yarn.js:446:10283)
Error: building at STEP "RUN yarn install && yarn build": while running runtime: exit status 1
```